dev/deps/spark-deps-hadoop-3.2-hive-2.3

```diff
@@ -10,6 +10,7 @@ annotations/17.0.0//annotations-17.0.0.jar
 antlr-runtime/3.5.2//antlr-runtime-3.5.2.jar
 antlr4-runtime/4.8-1//antlr4-runtime-4.8-1.jar
 aopalliance-repackaged/2.6.1//aopalliance-repackaged-2.6.1.jar
+arpack/1.3.2//arpack-1.3.2.jar
```

Comment:
There's one more minor thing to do here. Under licenses-binary/ , include a file like `LICENSE-blas.txt` and/or separate ones for arpack, lapack artifacts as you wish (name doesn't matter, doesn't have to be 3 - really it's whatever satisifies you, as long as we have a copy that's connectable to your artifacts) with the text of the MIT license file. In `LICENSE-binary`, you'll see a list of MIT dependencies. Just add them in there too.

I don't think any dependencies ultimately went away so there's nothing to remove, AFAICT.

It's a formality but just ensures the license requirements are buttoned up.

Reply:
Added with https://github.com/apache/spark/pull/32253/commits/17edfebb92586c77c8806f2a63daaa166f524657